docs/pages/enroll-resources/machine-id/troubleshooting.mdx

```diff
@@ -320,3 +320,18 @@ outputs:
 
 Failure to add the `disable_exec_plugin` flag will result in a warning being displayed:
 `Destination kubernetes_secret: identity-output must be a directory in exec plugin mode`.
+
+## Configuring `tbot` for split DNS proxies
+
+When you have deployed your Proxy Service in such a way that it is
+accessible via two different DNS names, e.g an internal and external address,
+you may find that a `tbot` that is configured to use one of these addresses may
+attempt to use the other address and that this may cause connections to fail.
+
+This is because `tbot` queries an auto-configuration endpoint exposed by the
+Proxy Service to determine the canonical address to use when connecting.
+
+To fix this, set a variable of `TBOT_USE_PROXY_ADDR=1` in the environment of the
```

Comment:
Other boolean environment variables like this (`TELEPORT_ENABLE_RDP_NLA`, `TELEPORT_UNSTABLE_ALLOW_OLD_CLIENTS` etc) use `yes` rather than `1`. I feel like it might be confusing to have a different format here?

Reply:
[c0107c5](https://github.com/gravitational/teleport/pull/48592/commits/c0107c502706f9e992f97e5876bf4a56cb40abc0)